package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "slimerjs-core",
+  "version": "1.0.0",
+  "description": "A scriptable browser like PhantomJS, based on Firefox",
+  "bin": {
+    "slimerjs-firefox": "./bin/slimerjs-core"
+  },
+  "main": "./lib/slimerjs-core.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/laurentj/slimerjs.git"
+  },
+  "scripts": {
+    "install": "node install.js"
```

Comment:
that was the entire point of why I didn't want to support slimerjs anymore. Because slimerjs is just a bit of bash/python/js around Firefox it doesn't need a platform specific downloader. It can be a plain npm package.

You should concentrate your efforts on porting https://github.com/laurentj/slimerjs/blob/master/src/slimerjs.py https://github.com/laurentj/slimerjs/blob/master/src/slimerjs or https://github.com/laurentj/slimerjs/blob/master/src/slimerjs.bat to slimerjs.js